Library/Homebrew/formula_installer.rb

```diff
@@ -593,17 +593,7 @@ def install
         pour
       # Catch any other types of exceptions as they leave us with nothing installed.
       rescue Exception # rubocop:disable Lint/RescueException
-        ignore_interrupts do
-          begin
-            FileUtils.rm_r(formula.prefix) if formula.prefix.directory?
-          rescue Errno::EACCES, Errno::ENOTEMPTY
-            odie <<~EOS
-              Could not remove #{formula.prefix.basename} keg! Do so manually:
-                sudo rm -rf #{formula.prefix}
-            EOS
-          end
-          formula.rack.rmdir_if_possible
-        end
+        Keg.new(formula.prefix).uninstall_ignore_interrupts_and_raise_failures! if formula.prefix.directory?
```

Comment:
Was this intended to no longer print the odie message as that only happens when `raise_failure: false`?

https://github.com/Homebrew/brew/blob/490e51d47fb5a60370b3549ebf4a4cfb6c2044a2/Library/Homebrew/keg.rb#L296-L301